rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -13,173 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
If the file doesn't exist, I think ppl will ask themselves questions, read at the git log and find out what happened.
Also they could ask around. Or check into user_osa_variables_defaults. Or read the release notes.

Plenty of sources of information, I don't see why we need to keep a file on top of it, just for devs.
I'd agree on users, but this file was never intended to touch by deployers.